package.json

```diff
@@ -87,6 +88,7 @@
     "react-native-image-picker": "^4.1.2",
     "react-native-image-size": "^1.1.3",
     "react-native-keyboard-spacer": "^0.4.1",
+    "react-native-mime-types": "^2.3.0",
```

Comment:
Hmmm?? Let's see if that is really needed.

Reply:
I had checked about this in the previous comments https://github.com/Expensify/App/pull/7849#issuecomment-1048174075

Reply:
Sorry but no! I don't think we should use this lib. we are only checking three common files Video, image, and other. May be try to pass down that mime-type from the file-uploader. 
![image](https://user-images.githubusercontent.com/24370807/159708837-e77a8911-6a41-4fae-b636-b62d69a90c61.png)

Def a No for this lib.